gpsSensor.py

```diff
@@ -16,30 +16,23 @@ def getLoc(log):
     try:
         while True:    
             line = gpsSerial.readline().decode('ascii', errors='replace').strip()
-
             # Check if the line contains a GLL sentence (you can also use GGA, RMC, etc.)
```

Comment:
Consider updating this comment in a future commit, but not worth redoing the RP.